src/Style/Style.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
```

Comment:
Well there is no class for now. But yes, I guess all the style should eventually be here.

But I have so far no plans to use that in the syntax highlighter.

Reply:
Sorry, I wanted to refer to the Viewer/Dialog in my question. I've edited accordingly.

Yes, my question is because if we don't have plans to include all the style I would prefer to have this class inside the CodeViewer folder.

Reply:
I don't understand. This code is coming from Orbit/main.cpp and will be used in both CodeViewerDemo and Orbit/main.cpp.

Reply:
Why do you want to put it into CodeViewer? From my understanding this has very little to do with CodeViewer.